### ISSUE TITLE
[Feature]: Set MTP status for chocolate models

### ISSUE THREAD

**Issue by SabinStargem** [NousResearch/hermes-agent#114108]
### Problem or Use Case

I am using a model called "Swift-Qwen3.8-27B-Uncensored-Dynamic-MTP-UD-Q6_K_XL".   Herme's model ID catalog doesn't recognize this model, thus MTP isn't enabled.

### Proposed Solution

It would be good if we can have the ability to enable and disable features like MTP on a per-model basis.

### Feature Type

Configuration option

### Scope

Medium (few files, < 300 lines)

### Contribution

- [ ] I'd like to implement this myself and submit a PR

### Debug Report (optional)

```shell

```

**Comment by kokhlo** [comment]
I dug into current main (`98f758ae`) and the catalog path explains exactly why your Swift-Qwen3.8 build never gets MTP.

**Diagnosis.** MTP capability is resolved exclusively from the curated catalog, `hermes_cli/local_runtime/presets.py:93`:

```python
is_mtp = entry.mtp if entry is not None else model_id in mtp_capable
```

For any staged GGUF whose model id has no catalog entry, `is_mtp` falls back to the `mtp_capable` set — and the only production caller, `bootstrap._generate_presets` (`hermes_cli/local_runtime/bootstrap.py:160`), never passes one, so that set is always empty. Downstream, `plan_launch` never applies the MTP memory posture (kv_scale 1.2 + the doubled logits buffers, `context_policy.py:118-133`) and `launch_args` never emits `--spec-type draft-mtp --backend-sampling` (`context_policy.py:235-238`). A dynamically-MTP'd build like yours is staged and launched exactly like a plain non-MTP file, silently.

**Plan.** A per-model MTP override, mirroring the existing window-override machinery (`growth.py:load_window_overrides`, consumed at `presets.py:99`):

- `mtp_overrides.json` next to the presets in `runtimes_root()` (machine-scoped, same rationale as the preset INI): `{model_id: bool}`. Explicit `true` enables MTP for out-of-catalog files; explicit `false` disables it on a catalog model whose draft acceptance you don't like.
- `preset_for_model` consults it: the override wins when present, catalog `entry.mtp` stays the default, and unknown models default to off — blindly turning on backend sampling for an arbitrary GGUF is an OOM risk, so this stays opt-in.
- A route on the local-models router to set/clear the override and regenerate presets (the same place `DELETE /models/{model_id}` already clears the window override).
- Tests: override enables the spec-decode launch args and MTP memory posture for an uncatalogued GGUF; explicit `false` suppresses them for a catalog MTP entry; catalog models are unchanged when no override exists.

PR within the hour.
